.github/workflows/galata.yml

```diff
@@ -49,6 +49,7 @@ jobs:
       - name: Install browser
         run: |
           cd galata
+          jlpm playwright install-deps || true
```

Comment:
Do we need `|| true`?

Reply:
That's a good question. I copied the line from the action in which I guess we want to be robust to the case the os is not well supported. I guess in this case we control the os, so it will be better to remove it.